.circleci/config.yml

```diff
@@ -93,11 +93,6 @@ jobs:
           name: Set BASH_ENV
           command: ./tools/circleci_bash_env.sh
 
-      - run:
-          name: check neuromag2ft
-          command: |
-            neuromag2ft --version
```

Comment:
I think actually CircleCI shouldn't need the minimal commands... pushed with `[circle full]` to double check